src/mqtt-broker/src/subscribe/sub_common.rs

```diff
@@ -132,13 +137,45 @@ pub fn decode_share_info(sub_name: String) -> (String, String) {
 pub async fn get_share_sub_leader(
     client_pool: Arc<ClientPool>,
     group_name: String,
-) -> Result<GetShareSubLeaderReply, MqttBrokerError> {
+) -> Result<GetShareSubLeaderReply, CommonError> {
```

Comment:
CommonError needs to be replaced with MqttBrokerError

Reply:
There are other places in mqtt broker where CommonError is used. Keep using CommonError for now, and then use MqttBrokerError to replace CommonError uniformly in the future